cockatrice/src/user_context_menu.h

```diff
@@ -27,10 +27,12 @@ class UserContextMenu : public QObject {
     QAction *aAddToIgnoreList, *aRemoveFromIgnoreList;
     QAction *aKick;
     QAction *aBan;
+    QAction *aPromoteToMod, *aDemoteFromMod;
```

Comment:
Not fully necessary, but do you want to put all the `QAction` vars into one line?